### ISSUE TITLE
FreeBSD port

### ISSUE THREAD

**Issue by woodsb02** [michael-yuji/xc#14]
To make xc more accessible to the average person running FreeBSD, it would help to create a port for it.

That way the installation steps could just include “pkg install xc” (plus other steps to enable linuxulator).

**Comment by elliptic-kitty** [comment]
@woodsb02 `xc` maintainer here, I'd love to have xc in ports, except it is not really ready yet as the interface hasn't be stabilized and we don't want users to have bad experience migrating version to version.  

Maybe a `xc-dev` port will be a good idea and warn users that backward compatibility is not guaranteed, until the 1.0 version eventually ready. This maybe a slow process (since the original author joined the fruit company) but hopefully we can get there soon.

**Comment by woodsb02** [comment]
Hey @elliptic-kitty, I personally don't think of an official port/package as any promise of stability or backwards compatibility. It just allows users easy access to install, use and test the software.

As for a "-dev" package, I only think those are required when you want to have 2 versions available for users to choose from (the stable release or the development branch).

As for pre-release / early development software, I would recommend just an xc port that includes a disclaimer in the description about the lack of garantees.

**Comment by davidchisnall** [comment]
I agree.  Ports all come with versions, if the version is 0.1 or whatever then it's not a 1.0 release.  You can always stick something in the post-install message that says 'command-line interface not stable until 1.0!'  I believe the Podman port contains a similar 'don't use this in production yet' disclaimer.

Ideally a port would also come with an rc script for running xcd, so that you just enable it with `sysrc` rather than needing to start it manually.
